modules/common/src/main/scala/io/circe/jackson/enrich/package.scala

```diff
@@ -0,0 +1,3 @@
+package io.circe.jackson
+
+package object enrich extends WithJacksonMapper with JacksonParser with JacksonCompat {}
```

Comment:
Can be `com.snowplowanalytics.snowplow.enrich.common.utils.CirceUtils.parser`, again just to make it clear why we need this.